x-pack/plugins/apm/public/components/shared/charts/Legend/index.js

```diff
@@ -16,7 +16,6 @@ const Container = styled.div`
   cursor: ${props => (props.clickable ? 'pointer' : 'initial')};
   opacity: ${props => (props.disabled ? 0.4 : 1)};
   user-select: none;
-  padding-bottom: 15px;
```

Comment:
If we need to remove this, we need a different way to avoid the issue when legend items spill over to a 2nd line.

<img width="620" alt="screen shot 2019-01-02 at 8 51 47 am" src="https://user-images.githubusercontent.com/159370/50594776-e7842f00-0e6b-11e9-801e-43e750d7083b.png">

Reply:
That sounds like the responsibility of the `Legends` container. 
`Legend` is a reusable component and adding padding causes issues in the other places where it's used.

Reply:
For instance, see how the agent marker points is lifted above the line, and how Services is not aligned with the legends:

<img width="853" alt="screenshot 2019-01-02 at 15 08 18" src="https://user-images.githubusercontent.com/209966/50595373-4e234000-0ea0-11e9-8123-602895e5a803.png">

Reply:
@jasonrhodes I've added a commit that should fix this.